hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieParquetInputFormat.java

```diff
@@ -63,6 +62,7 @@
  * that does not correspond to a hoodie table then they are passed in as is (as what FileInputFormat.listStatus()
  * would do). The JobConf could have paths from multipe Hoodie/Non-Hoodie tables
  */
+@UseRecordReaderFromInputFormat
```

Comment:
Is this the main change ?

Reply:
Yes he main change required is this to integrate with presto. Rest are just fixes for the various runtime issues I ran into trying to get this working.